Flush support for Python and .NET SDK (#402)
rudderlabs/rudderstack-docs: stream-sources/rudderstack-sdk-integration-guides/rudderstack-dotnet-sdk.mdx

| `timestamp` | Date | Optional | The timestamp of the message's arrival. |
| `context` | Object | Optional | Dictionary of information that provides context about a message. However, it is not directly related to the API call. |

## Flushing events

> To flush your events on demand and make sure no events are left in the queue, you can use the SDK's `flush()` method.

Reviewer: ```suggestion
To make sure no events are left in the queue, you can flush the events explicitly by using the SDK's `flush()` method.
```
